packages/angular_devkit/build_angular/src/builders/ssr-dev-server/index.ts

```diff
@@ -80,7 +80,18 @@ export function execute(
     verbose: options.verbose,
   } as json.JsonObject);
 
-  const bsInstance = require('browser-sync').create();
+  let browserSync: typeof import('browser-sync');
+  try {
+    browserSync = require('browser-sync');
+  } catch {
+    return {
+      success: false,
+      error:
+        '"browser-sync" is not installed, most likely you need to run `npm install browser-sync --save-dev` in your project.',
+    };
```

Comment:
Can you wrap this object literal in an rxjs `of` call to keep the API signature?

Reply:
Done